db/user.go

```diff
@@ -0,0 +1,81 @@
+package db
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+var ErrNoUser = errors.New("no user")
+
+type User struct {
+	Id    int    `json:"id"`
+	Email string `json:"email"`
+	Token string `json:"token"`
+}
+
+func FindUserBy(column string, value interface{}) (*User, error) {
```

Comment:
I feel like you should abstract this whole thing into db.go. As in make a generic function for getting x model by y attribute.